files/galaxy/tpv/interactive_tools.yml

```diff
@@ -6,7 +6,6 @@ tools:
     params:
       docker_volumes: $defaults
       container_monitor_result: callback
-      submit_requirements: 'GalaxyDockerHack == True && GalaxyGroup == "interactive"'
```

Comment:
This should exist for all other users not part of the `oubb` role, or am I missing something?

Reply:
sorry I copied the code from your snipped from the wrong part